contrib/linearize/README.md

```diff
@@ -46,7 +46,7 @@ linearize-hashes.py.
 (Default: `1000*1000*1000 bytes`)
 * `netmagic`: Network magic number.
 * `out_of_order_cache_sz`: If out-of-order blocks are being read, the block can
-be written to a cache so that the blockchain doesn't have to be seeked again.
```

Comment:
I would keep this as-is, since "seek" has a technical meaning (referring to the system call of the same name). It's not necessarily grammatical English, but I'd consider it grammatical programmer-English.